### PR TITLE
fire beacon event on logout

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

**Comment by aintgoin2goa** [comment]
@commuterjoy   @keirog

File: src/app.js [hunk 1]
```diff
@@ -28,6 +47,7 @@ app.get('/logout', function (req, res) {
 	var redirectUrl = req.get('Referrer') || '/';
 	res.set('Cache-Control', 'private, max-age=0, no-cache');
 	res.clearCookie('FTSession', {domain:'.ft.com', path:'/'});
+	fireBeacon('logout', req);
```
Comment (keirog) [inline src/app.js]: Probably a good idea to fire it after the response

**Comment by keirog** [comment]
Not sure about adding the beacon url to makefile but otherwise :+1:

**Comment by aintgoin2goa** [comment]
If I don't `make run` will fail I can't test locally - is it really that much a of secret?

**Comment by keirog** [comment]
Nah, just another place to update it if it changes. I guess really the node client should just log a warning something and return, rather than throw an error.

**Pull request merged**